charts/ozone/templates/ozone-configmap.yaml

```diff
@@ -19,7 +19,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ .Release.Name }}-ozone
```

Comment:
Why did you changed ConfigMap name?

Reply:
I changed it because I used ozone helm chart as dependency chart. When using plain release name, I got a conflict with an already existing configMap of the main chart with `{{ .Release.Name }}`. This could avoid such conflicts and, in my opinion, this can help to identify the belonging of the configMap faster in such aggregated charts. But it is definitely not needed. I can change this back if you prefer it :)